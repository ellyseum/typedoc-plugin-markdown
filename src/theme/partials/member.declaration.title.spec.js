describe(`member.declaration.title.hbs`, () => {
  test('should compile', () => {
    const context = {
      name: 'numbers',
      originalName: 'numbers',
      kind: 32,
      defaultValue: ' [1, 2, 3]',
      type: { type: 'array', elementType: { type: 'intrinsic', name: 'number' } },
      kindString: 'Variable',
    };
    const result = global.compileTemplate('member.declaration.title.hbs', context);
    expect(result).toMatchSnapshot();
  });
});
