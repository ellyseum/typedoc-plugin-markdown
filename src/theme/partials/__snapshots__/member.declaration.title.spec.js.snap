// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`member.declaration.title.hbs should compile 1`] = `"**‚óè numbers**: *[type]* = [1, 2, 3]"`;
